Britannia/common/buildings/BTWK_special_buildings.txt

```diff
@@ -177,24 +177,122 @@ offas_dyke_01 = {
 ##################
 # Dumbarton Rock #
 ##################
-btwk_rock_of_dumbarton_01 = {  # Add upgrades to this?
+btwk_dumbarton_rock_01 = {   # Could do with a hill-esque icon 
+	construction_time = slow_construction_time
+	type_icon = "icon_structure_stonehenge.dds"
+
+	can_construct = {
+		culture = { 
+			has_innovation = innovation_crop_rotation 
+			culture_is_brythonic_culture_trigger = yes
+		}
+	}
+
+	cost_gold = expensive_building_tier_3_cost
 
+	province_modifier = {
+		fort_level = 2
+		defender_holding_advantage = 10
+		station
```

Comment:
Unfinished line

Reply:
Was going to put stationed archers toughness (i.e. fort is elevated on the rock => high ground advantage) - sound okay?   I know that they wouldn't have that "high ground" in battle elsewhere, just thought it might be an interesting modifier to throw in given the context

Reply:
Think that's fine - alternative I guess would be whatever cultural MAA Cumbrians have (do they even have one?)

Reply:
No they don't